src/contractor/contractor.cpp

```diff
@@ -299,18 +300,17 @@ parse_segment_lookup_from_csv_files(const std::vector<std::string> &segment_spee
         const auto file_id = idx + 1; // starts at one, zero means we assigned the weight
         const auto filename = segment_speed_filenames[idx];
 
-        std::ifstream segment_speed_file{filename, std::ios::binary};
-        if (!segment_speed_file)
-            throw util::exception{"Unable to open segment speed file " + filename};
-
+        storage::io::FileReader segment_speed_file_reader(std::string(filename),
+                                                          storage::io::FileReader::HasNoFingerprint);
+       
         SegmentSpeedSourceFlatMap local;
 
         std::uint64_t from_node_id{};
         std::uint64_t to_node_id{};
         unsigned speed{};
 
-        for (std::string line; std::getline(segment_speed_file, line);)
-        {
+        std::for_each(segment_speed_file_reader.GetLineIteratorBegin(), segment_speed_file_reader.GetLineIteratorEnd(), [&](const std::string &line) {
```

Comment:
clang-format-3.8 all files touched by this changeset - check the scripts directory for a format.sh script